test/integration/kubernetes_deploy_test.rb

```diff
@@ -543,17 +543,28 @@ def test_deploy_result_logging_for_mixed_result_deploy
       %r{ConfigMap/test\s+Available}
     ], in_order: true)
 
+    if KUBE_SERVER_VERSION < Gem::Version.new("1.10.0")
+      start_bad_probe_logs = [
+        %r{Deployment/bad-probe: TIMED OUT \(timeout: \d+s\)},
+        "Timeout reason: hard deadline for Deployment"
+      ]
+      end_bad_probe_logs = [%r{Unhealthy: Readiness probe failed: .* \(8 events \)}] # event
```

Comment:
I think its ok to ignore Policial here since we've been using the `%r{}` syntax for regex and I don't think we want to start using the `//` syntax.

Reply:
The rule it is enforcing is the "mixed" style if you look at the rubocop cache file. If we don't want that, we need to change our rubocop rules. But in general I'm inclined to stick with the standard style guide.